.github/workflows/dojo-e2e.yml

```
@@ -0,0 +1,114 @@
+      - name: Save E2E URLs
+        working-directory: examples/e2e
+        run: |
+          echo "${{ inputs.urls }}" | base64 -d > app-configs.json
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)